rosidl_generator_c/package.xml

```diff
@@ -7,11 +7,11 @@
   <maintainer email="william@osrfoundation.org">William Woodall</maintainer>
   <license>Apache License 2.0</license>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <build_depend>rosidl_typesupport_interface</build_depend>
 
-  <buildtool_export_depend>ament_cmake</buildtool_export_depend>
+  <buildtool_export_depend>ament_cmake_ros</buildtool_export_depend>
```

Comment:
I incorrectly assumed that ament_cmake_ros had to be exported to other packages, but it doesn't seem to be the case. Removed in https://github.com/ros2/rosidl/pull/183/commits/f6bc5200d3ecf10129d8d81283822614516d3a67